docs/developers/bot-param.md

```diff
@@ -36,9 +38,34 @@ All three bots (liquidator, disputer, and monitor) share some configuration opti
 - `PAGERDUTY_FROM_EMAIL`[optional] Each Pagerduty service also requires a `from email` to uniquely identify the logger.
 - `INFURA_API_KEY`[optional]: Override the default Infura key used by the bot.
 
+## Optimistic Oracle bot
+
+The `optimistic-oracle` bot proposes and disputes prices with the Optimistic Oracle based on off-chain information about the value of the price identifier. By default, the bot will only dispute prices that are off (according to its calculations) by more than 5%.
+
+## Minimum viable optimistic-oracle config:
+
+You can run an `optimistic-oracle` bot by providing a mnemonic with an associated private key that the bot should use and a `COMMON_PRICE_FEED_CONFIG`. No financial product contract addresses are required, since the bot will watch for all active price requests in the Optimistic Oracle.
```

Comment:
```suggestion
You can run an `optimistic-oracle` bot by providing a mnemonic with an associated private key that the bot should use and a `COMMON_PRICE_FEED_CONFIG` to provide price feeds with api keys. No financial product contract addresses are required, since the bot will watch for all active price requests in the Optimistic Oracle.
```